modules/GraphvizLayout/src/main/resources/org/icculus/chunky/gephigraphviz/Bundle.properties

```diff
@@ -0,0 +1,45 @@
+# To change this template, choose Tools | Templates
+# and open the template in the editor.
+OpenIDE-Module-Display-Category=Layout
+OpenIDE-Module-Long-Description=\
```

Comment:
Please put everything on one line and remove the "\". Otherwise it won't look good on the marketplace.